test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
This file is not supposed to be seen on your repository. It will be fine if you add it to your gitignore file.